### PR TITLE
SQL: aggregation examples' tests

### PR DESCRIPTION
**Pull request by unrealwork**
## SQL: aggregation examples' tests
- [x] Tests

### CODE REVIEW

**Comment by rmakulov** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/sql/examples/aggregation/SqlExampleAggregateMaxValueTimeTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    private static final String TEST_METRIC_NAME = TEST_PREFIX + "metric";
+    private static final String TEST_ENTITY1_NAME = TEST_PREFIX + "entity-1";
+    private static final String TEST_ENTITY2_NAME = TEST_PREFIX + "entity-2";
+
+    @BeforeClass
+    public void prepareData() throws IOException {
+        List<Series> seriesList = Arrays.asList(
+                new Series() {{
+                    setMetric(TEST_METRIC_NAME);
+                    setEntity(TEST_ENTITY1_NAME);
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/examples/aggregation/SqlExampleAggregateMaxValueTimeTest.java]: don't forget to use registry

File: src/test/java/com/axibase/tsd/api/method/sql/examples/aggregation/SqlExampleSlidingWindowsTest.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    private final static String TEST_PREFIX = "sql-example-sliding-windows-";
+    private static final String TEST_METRIC_NAME = TEST_PREFIX + "metric";
+    private static final String TEST_ENTITY1_NAME = TEST_PREFIX + "entity-1";
+    private static final String TEST_ENTITY2_NAME = TEST_PREFIX + "entity-2";
+
+    @BeforeClass
+    public static void prepareData() throws IOException {
+        SeriesMethod.insertSeriesCheck(
+                Arrays.asList(
+                        new Series() {{
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/examples/aggregation/SqlExampleSlidingWindowsTest.java]: registry

**Comment by Sild** [comment]
review done

**Pull request merged**